content/en/blog/2022/exponential-histograms.md

```diff
@@ -0,0 +1,297 @@
+```
+
+The SDK provides implementations of meter provider, meter, and instruments. It
+aggregates measurements reported by instruments and exports them as metrics
+according to the application configuration.
+
+There are currently six types
+of [instruments](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument)
+in OpenTelemetry metrics: counter, up down counter, histogram, async counter,
+async up down counter, and async gauge. Carefully consider which instrument type
+to select, since each implies certain information about the nature of the
+measurements it records and how they are analyzed. For example, use a counter
+when you want to count things and when the sum of the things is more important
+than their individual values (such as tracking the number of bytes sent over a
+network). Use a histogram when the distribution of measurements is relevant for
+analysis. For example, a histogram is a natural choice for tracking response
+times for HTTP servers, because it's useful to analyze the distribution of
+response times to evaluate SLAs and identify trends. To learn more, see the
+guidelines
+for [instrument selection](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/supplementary-guidelines.md#instrument-selection).
+
+I mentioned earlier that the SDK aggregates measurements from instruments. Each
+instrument type has a default aggregation strategy (or
+simply [aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#aggregation))
+that reflects the intended use of the measurements as implied by the instrument
+type selection. For example, counters and up down counters aggregate to a sum of
+their values. Histograms aggregate to a histogram aggregation. (Note that
+histogram is both
+a [type of instrument](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#histogram)
+and
+an [aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#histogram-aggregations).)
+
+## Anatomy of a histogram
+
+What is a histogram? Putting OpenTelemetry aside for a moment, we're all
+somewhat familiar with histograms. They consist of buckets and counts of
+occurrences within those buckets.
+
+For example, a histogram could track the number of times a particular number was
+rolled with the sum of two six-sided dice, with one bucket for each possible
+outcome, from 2-12. Over a large number of rolls, you expect the 7 bucket to
+have the highest count because you are more likely to roll a combined total of
+7, and the 2 and 12 buckets to have the least because these are the least likely
+rolls, as shown in this example histogram.
+
+![histogram-outcomes-200-rolls-two-6-sided-dice](/img/blog-exponential-histogram/histogram-outcomes-200-rolls-two-6-sided-dice.webp)
```

Comment:
Please create a page bundle for this post. That is:

- Create a directory named `content/en/blog/2022/exponential-histograms`
- Rename `content/en/blog/2022/exponential-histograms.md` to `content/en/blog/2022/exponential-histograms/index.md`
- Move all images to be under under `content/en/blog/2022/exponential-histograms` and adjust the image paths to be local.

Let me know if you feel like you need more guidance or would like for me to help.